common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

```diff
@@ -1,24 +1,68 @@
 ---
 metadata:
-    display_name: Checkboxes
+    display_name: Checkbox
     markdown: |
-      A checkboxes problem presents checkbox buttons for student input. Students can select more than one option presented.
-      >>Select the answer that matches<<
+       Studio offers three types of problems that allow learners to select one or more options from a list that you provide.
+       - Checkbox problems allow learners to select multiple options. Learners can see all the options along with the problem text.
```

Comment:
Reviewers, sorry about the scrolling issues. The problem is that if I do the usual line wrap in the .rst file and paste the text into the simple editor, the line wraps in the simple editor match the ones in the .rst file, which is messy. I assume the templates work the same way, though I could be wrong about that.